sdk/go/pulumi/log.go

```diff
@@ -36,6 +37,7 @@ type Log interface {
 type logState struct {
 	engine pulumirpc.EngineClient
 	ctx    context.Context
+	join   *sync.WaitGroup
```

Comment:
Maybe I missed it, but where is this used?

Reply:
Good catch. I added this to ensure that we don't drop logs, but forgot to write the code to use it. Done.